apps/icelandic-names/backend/src/app/modules/icelandic-name/icelandic-name.controller.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Delete,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+} from '@nestjs/common'
+import { ApiNoContentResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger'
+
+import { IcelandicNameService } from './icelandic-name.service'
+import { IcelandicName } from './icelandic-name.model'
+import { UpdateIcelandicNameBody, CreateIcelandicNameBody } from './dto'
+
+@Controller('api')
+@ApiTags('icelandic-names')
+export class IcelandicNameController {
+  constructor(private readonly icelandicNameService: IcelandicNameService) {}
+
+  @Get('icelandic-names/all')
```

Comment:
There is no need to declare endpoints function in the path since we are using http methods in line with [RESTful]( https://restfulapi.net/resource-naming/) api concept.
Consider removing the path postfixes.
```suggestion
  @Get('icelandic-names')
```